src/common/prntbase.cpp

```diff
@@ -1976,6 +1982,18 @@ void wxPrintPreviewBase::InvalidatePreviewBitmap()
 
 bool wxPrintPreviewBase::UpdatePageRendering()
 {
+    static double s_sf = 0.0;
```

Comment:
Sorry, why is this static? It's possible to have different scale factors for different windows, so it should be a member variable, AFAICS.

Reply:
True. But I don't know if it will break ABI.

Reply:
In master we don't care about ABI. In 3.2, we need use the standard "emulation" of member variables which consists in adding a hash map with `this` as key and this value in it.